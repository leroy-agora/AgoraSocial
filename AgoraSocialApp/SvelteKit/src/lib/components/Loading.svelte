<div class="fixed h-screen w-screen top-0 left-0 flex items-center justify-center">
  <div class="animate-pulse text-purple-600 text-9xl">
    &#9830;
  </div>
</div>
