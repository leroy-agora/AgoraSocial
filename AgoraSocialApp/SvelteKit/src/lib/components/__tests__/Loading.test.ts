import { render } from '@testing-library/svelte';
import Loading from '../Loading.svelte';

describe('Loading Component', () => {
  it('should render', () => {
    const { container } = render(Loading);

    expect(container).toMatchSnapshot();
  });
  it('should show the loading indicator', () => {
    const { container } = render(Loading);
    expect(container).toHaveTextContent('â™¦');
  });
});
