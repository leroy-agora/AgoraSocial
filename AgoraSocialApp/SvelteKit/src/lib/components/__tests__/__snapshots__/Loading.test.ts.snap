// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render 1`] = `
<body>
  <div>
    <div
      class="fixed h-screen w-screen top-0 left-0 flex items-center justify-center"
    >
      <div
        class="animate-pulse text-purple-600 text-9xl"
      >
        â™¦
      </div>
    </div>
  </div>
</body>
`;
